src/styles/utils/available.scss

```diff
@@ -0,0 +1 @@
+$shift: 50%;
```

Comment:
why does this file available?
I think it is variable

Reply:
Yeah, it should've been variable